src/lib/polyfills/index.js

```diff
@@ -0,0 +1,2 @@
+import 'babel-polyfill';
```

Comment:
I wonder if we really need to import the whole polyfill, or just the parts that we need? For right now we should probably just merge as-is, though.

Reply:
Babel only includes what is needed from the code, why would we try to determine that ourselves. right?

Reply:
I'm still thinking about bundle size, not runtime checking.

Reply:
I don't think it will change the determination for bundle size in the end.

Reply:
> Babel only includes what is needed from the code, why would we try to determine that ourselves. right?

If you set `useBuiltIns: true` above, then yes, that is true.